documentation/contracts.md

```diff
@@ -224,4 +224,4 @@ The Feather sidecar SHOULD NOT require the `jp-user-id` and `jp-user-role` heade
 
 ## Environment variables
 
-TODO
+The `PORT` environment variable SHOULD be supported by all services. The default value for the port SHOULD be defined by the service itself. The service SHOULD listen on the configured port.
```

Comment:
In Spring Boot, the environment variable for this is called 'SERVER_PORT', and while there is certainly a workaround to react to another environment variable, Im not convinced I should change that. I don't know how difficult it is to change it in your case, but if it's easy and you'd like to do it then I would suggest just that name, otherwise I would put the name of the specific environment variable as service specific